doc/book/service-manager.md

```diff
@@ -73,6 +82,21 @@ the configuration (`MyLogger`):
 $logger = $container->get('MyLogger');
 ```
 
+Notes:
+
+- The keys of the writers are not required, but they make the merge of the
```

Comment:
> …they make the merge of the config files easier.

For whom? For the script or user or something else? Very confusing!

Reply:
Ok, I remove this part.